### PR TITLE
Small cleanup to wait_test.go

### PR DESCRIPTION
**Pull request by gmarek**
cc @wojtek-t

### CODE REVIEW

File: pkg/util/wait/wait_test.go [hunk 1]
```diff
@@ -123,10 +123,11 @@ func TestPollImmediate(t *testing.T) {
 		invocations++
 		return true, nil
 	})
-	fp := fakePoller{max: 0, wg: sync.WaitGroup{}}
+	fp := fakePoller{max: 0}
 	if err := pollImmediateInternal(fp.GetWaitFunc(), f); err != nil {
 		t.Fatalf("unexpected error %v", err)
 	}
+	// We don't need to wait for fp.wg, as pollImmediate shouldn't call WaitFunc at all.
```
Comment (fgrzadkowski) [inline pkg/util/wait/wait_test.go]: If `pollImmediateInternal` is not supposed to call WaitFunc then why we pass it there?

Comment (gmarek) [inline pkg/util/wait/wait_test.go]: It depends on what kind of error do we want to get if it'll get called - panic, or test failure. I can pass nil there if you prefer.

**Comment by mwielgus** [comment]
LGTM

**Comment by gmarek** [comment]
@k8s-bot unit test this

**Comment by gmarek** [comment]
ERROR: Error fetching remote repo 'origin'
@k8s-bot unit test this

**Pull request merged**